### DISCUSSION TITLE
Как запускать проект не в консоли?

### DISCUSSION THREAD

**Discussion by SaBond0** [AvaloniaUI/Avalonia#14206] [Q&A]
проект как-будто запускается в консоли, т.к. уменьшаю размер формы и текста почти не видно, а при запуске все видно и цвет текста и фона, обратно пропорционален, от тех которые видны на форме в visual studio

**Comment by thevortexcloud** [comment]
Please provide your code. It should open an Avalonia window, not a console one.

**Reply by SaBond0** [reply]
i he write this ? i don't you,specifically please

**Reply by thevortexcloud** [reply]
I am not sure what this is meant to mean.

**Reply by SaBond0** [reply]
What do I need to do?), My english Sorry )

**Reply by thevortexcloud** [reply]
Please show me your main method and `App.xaml.cs` file.

**Reply by SaBond0** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/156509288/e9de3f19-fd4c-4c36-8c8a-037320e842f7)

**Reply by thevortexcloud** [reply]
I think I have misunderstood the problem. It sounds like the problem is the window is not opening to the correct size?

**Reply by SaBond0** [reply]
I have the form itself, it opens in the console, and I want it to open in a separate form, of its own size and not the size of the console

**Reply by thevortexcloud** [reply]
The size of a window is controlled by the `Width` and `Height` properties, you can set that in the XAML of the window. If you want a different window you need to instantiate one and call `Show()` on it. There are some examples here: https://docs.avaloniaui.net/ru/docs/reference/controls/window

However I think you might be referring to content of the window, in which case you just need to use some sort of layout control as your root control in your window. EG `StackPanel`, `Grid`, etc. There are examples here (which seem to be in Russian): https://docs.avaloniaui.net/ru/docs/get-started/test-drive/add-some-layout
